modules/cluster-logging-release-notes-5.0.2.adoc

```diff
@@ -0,0 +1,9 @@
+[id="cluster-logging-release-notes-5-0-2"]
+= OpenShift Logging 5.0.2
+
+This release includes Red Hat Bug Advisory, link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 for OpenShift Logging Bug Fix Release (5.0.1)].
+
+[id="openshift-logging-5-0-2-bug-fixes"]
+== Bug fixes
+
+* If you did not set `.proxy` in the cluster installation configuration and then configured a global proxy on the installed cluster, a bug prevented Fluentd from forwarding logs to Elasticsearch. To work around this issue, in the proxy/cluster configuration, set `no_proxy` to `.svc.cluster.local` so it skips internal traffic. The current release fixes the proxy configuration issue. Now, if you configure the global proxy after installing an OpenShift cluster, Fluentd forwards logs to Elasticsearch. (link:https://issues.redhat.com/browse/LOG-1187[*LOG-1187*])
```

Comment:
* Just wondering whether anything in "proxy/cluster configuration" should be literal in backticks. 
* an OpenShift cluster -> an {product-title} cluster

Reply:
Thanks. Updated. I don't see any items that need backticks.